google/resource_container_cluster_test.go

```diff
@@ -291,6 +317,7 @@ func testAccCheckContainerCluster(n string) resource.TestCheckFunc {
 			{"description", cluster.Description},
 			{"endpoint", cluster.Endpoint},
 			{"instance_group_urls", igUrls},
+			{"legacy_abac.0.enabled", strconv.FormatBool(cluster.LegacyAbac.Enabled)},
```

Comment:
I don't believe this is accurately testing this as the code stands. We can't send false, so we never actually change the value. The test doesn't seem to verify that the value is now false.